solution/ui/regulations/eregs-vite/src/views/PolicyRepository.vue

```diff
@@ -91,27 +117,16 @@ getDocList();
             </HeaderComponent>
         </header>
         <div id="policyRepositoryApp" class="repository-view">
-            <template v-if="policyDocList.loading">
+            <template v-if="policyDocList.loading || partsLastUpdated.loading">
                 <p>Loading...</p>
             </template>
-            <template v-else-if="policyDocList.results.length > 0">
-                <div
-                    v-for="doc in policyDocList.results"
-                    :key="doc.uid"
-                    class="doc-list__item"
-                >
-                    <p>Name: {{ doc.name }}</p>
-                    <p>Description: {{ doc.description }}</p>
-                    <p>UID: {{ doc.uid }}</p>
-                    <a :href="getDownloadUrl(doc.uid)">Download</a>
-                    <p>Related Citations:</p>
-                    <p v-for="loc in doc.locations">
-                        {{ loc.title }} CFR § {{ loc.part }}.{{ loc.section_id }}
-                    </p>
-                </div>
-            </template>
             <template v-else>
-                <p>No results found.</p>
+                <PolicyResults
+                    :base="homeUrl"
+                    :results="policyDocList.results"
+                    :parts-last-updated="partsLastUpdated.results"
```

Comment:
unfortunately Vue 2 does not make it easy to use provide/inject for reactive props (like `parts-last-updated` here, which begins as an empty object and is updated with the result of an API call), so this still has to be drilled down through the many descendant components to get to where it is needed.

This is something that might be able to be corrected with a dedicated story.  Vue3 (when we upgrade) should also make this much easier, so I think this should be addressed once we upgrade.